volatility/framework/interfaces/objects.py

```diff
@@ -176,6 +176,32 @@ def has_member(self, member_name: str) -> bool:
         """
         return False
 
+    def has_valid_member(self, member_name: str) -> bool:
+        """Returns whether the dereferenced type has a valid member."""
+        if self.has_member(member_name):
+            try:
+                _ = getattr(self, member_name)
+                return True
+            finally:
```

Comment:
Putting a return False in a finally block means that this part of the function will always return False, even if the return True is hit in the try block. I tested and this is definitely the case, even for members that are in the structure. This link explains why: https://stackoverflow.com/a/19805746

Reply:
@ikelos  tagging you here as I made 3 comments on this and want to make sure you see them. I am not sure if GitHub sends emails if you aren't tagged.

Reply:
I'm the assignee, so I got the emails, but also glad to be tagged so I spot them.  5:)

Interesting, I guess it makes sense but I hadn't realized the python flow control worked that way.  I find a way to avoid `except Exception` using `contextlib.suppress`, so I'll add that to this branch.